ui/src/router/components/form/components/experiment_config/ExperimentConfigPanel.js

```diff
@@ -39,6 +41,7 @@ const CustomExperimentEngineConfigGroup = ({
           fallback={<FallbackView text="Loading Experiment Engine config" />}
           projectId={projectId}
           config={config}
+          protocol={protocol}
```

Comment:
Could you help update the doc for the experiment engine developer guide? Particularly, the UI contract for the custom experiment engine - adding the protocol argument to the `EditExperimentEngineConfig` and `ExperimentEngineConfigDetails` components: https://github.com/caraml-dev/turing/blob/main/engines/experiment/docs/developer_guide.md#ui-1